### ISSUE TITLE
Document accessibility considerations for Custom Highlight API

### ISSUE THREAD

**Issue by MoritzGlantz** [mdn/content#43408]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/CSS_Custom_Highlight_API

### What specific section or headline is this issue about?

Missing Accessibility section

### What information was incorrect, unhelpful, or incomplete?

The site should have a paragraph about how the API is supported by assistive technology, like many other MDN pages (e.g. [`<mark>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/mark#accessibility)). Correct me if I'm wrong, but as of right now there does not seem to be much information about the support of assistive technology with the Custom Highlight API (besides a [CSS wg thread from 2021](https://github.com/w3c/csswg-drafts/issues/6498)). It is important to add this kind of information, because people using this API might not be aware of the implications using this, since from the outside it could look like it is well supported. There is no mention what the "[type](https://drafts.csswg.org/css-highlight-api-1/#highlight-types)" is supposed to do and how well it is supported in screen readers.

### What did you expect to see?

An explanation of the current state of accessibility of this API.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/css_custom_highlight_api`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/CSS_Custom_Highlight_API
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/css_custom_highlight_api/index.md
* Last commit: https://github.com/mdn/content/commit/85fccefc8066bd49af4ddafc12c77f35265c7e2d
* Document last modified: 2025-11-07T15:58:06.000Z

</details>

**Comment by mohammed-ali-osman** [comment]
I noticed the [CSS Custom Highlight API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Custom_Highlight_API) page currently has no accessibility guidance. For HTML elements, MDN usually has an inline Accessibility section, but I’m not sure if this is the right pattern for a Web API page.

Should I add a ``## Accessibility`` section inline (after “Concepts and usage”), or would it be better to reference accessibility guidance elsewhere?

**Comment by MoritzGlantz** [comment]
Sorry for not replying earlier. I think it would be fine on the page itself.

**Comment by mohammed-ali-osman** [comment]
No, you don’t need to be sorry. I left that comment so an MDN member could see it and give me some suggestions.
